.tekton/push.yaml

```diff
@@ -15,15 +15,11 @@ spec:
       value: "{{ revision }}"
     - name: output-image
       value: 'quay.io/redhat-appstudio/release-service:{{ revision }}'
-    - name: infra-deployment-update-script
```

Comment:
We don't want to update infra-deployments with the latest version of release-service anymore?

Reply:
we will. but it will now be taken care of by https://github.com/redhat-appstudio/release-service-catalog/tree/main/pipelines/rhtap-service-push